src/main/java/com/nedap/openehr/lsp/aql/AQLStorage.java

```diff
@@ -139,8 +139,7 @@ private AQLDocument createAQLDocument(String uri, String text) {
     }
 
     public Hover getHover(HoverParams hoverParams) {
-        String uri = hoverParams.getTextDocument() == null ?
```

Comment:
the deprecated getUri() of the 1.0 protocol version is now removed from the library. The getTextDocument() parameter is marked @NonNull, so this is safe to do.